snuba/clickhouse/query_dsl/accessors.py

```diff
@@ -134,7 +134,8 @@ def get_time_range_expressions(
 
 
 def get_time_range(
-    query: ProcessableQuery[Table], timestamp_field: str
+    query: Union[ProcessableQuery[Table], ProcessableQuery[Entity]],
```

Comment:
I don't think mypy runs on these feature branches, so I would run it manually to make sure this works.

Reply:
It does, it is a pre-commit hook

Reply:
The pre-commit hook I think only runs on the changed files. I would run it manually on the whole codebase just in case.